### DISCUSSION TITLE
Spirit Runeword

### DISCUSSION THREAD

**Discussion by Justeen208** [fabd/diablo2-runewizard#42] [3. BUGS & DATA CORRECTIONS]
I didn't know that Spirit runeword is a ladder-only Runeword, its should have the "L" mark on the runewizard, since I thought it can be used for Single Player but only to be surprised it doesn't since Runewizard has no indication that its a Ladder runeword

**Comment by fabd** [comment]
YEah my bad it was actually makerd as ladder and I see a previous update for D2R removed it. I think the reason is that those are non-ladder on D2R since D2R has newer ladders. I'll have to think of a way to keep Runewizard working for both OG and D2R. Perhaps adding an orange Ladder icon that means "Ladder-Only in D2LoD"

**Reply by Justeen208** [reply]
Yea that would be a better indicator and should be easy to distinguish
